test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Great job, @joramot :clap: However, this file (`test.md`) is not supposed to be in your file directories as specified by the requirement. While you have it included in your `.gitignore` file, it seems you have saved the commit to memory before ignoring it in the .gitignore.

![image](https://user-images.githubusercontent.com/99715640/226762154-f3d53654-1e2a-49a2-a7a9-4e08395caa96.png)


- To undo this, kindly run this command from your command line:
```
git rm --cached test.md
```
That will remove the file from your cached memory

Reply:
ok, thanks, I will fix the issue